core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationServiceSpec.scala

```diff
@@ -526,4 +507,71 @@ class GoogleProjectRegistrationServiceSpec
 
   }
 
+  it should "throw an error if updating the billing account fails" in {
+    val mockSamDAO = mock[SamDAO]
+    val mockGoogleProjectRegRepo = mock[GoogleProjectRegistrationRepository]
+    val mockContext = mock[RawlsRequestContext]
+    val mockBillingRepository = mock[BillingRepository]
+    val mockGoogleServicesDAO = mock[GoogleServicesDAO]
+
+    when(mockGoogleServicesDAO.setBillingAccountName(any[GoogleProjectId], any[RawlsBillingAccountName], any()))
+      .thenThrow(new RuntimeException("Something has gone wrong in Google"))
```

Comment:
Since `setBillingAccountName` returns a `Future`, this needs to be

`.thenReturn(Future.failed(new RuntimeException("Something has gone wrong in Google")))`